keymanapp/keyman
# chore(common): Check in crowdin strings for Marghi 📜

This checks in the crowdin strings for the locale `mrt` for Marghi. Does not include macOS.

- [x] TODO: @sgschantz to check on adding mrt (mrt-NG?) locale in XCode

## User Testing
For the listed platforms:
1. Load the PR build
2. Set the UI/locale to Marghi
3. Verify the UI is Marghi
Note: The crowdin translator left some strings in English, so that's out of our control

* **TEST_ANDROID**
1. Verify Marghi strings are like the screenshot
![marghi](https://user-images.githubusercontent.com/7358010/146138491-3b02578d-7be1-4e82-8141-e4819e03fcbe.png)

* **TEST_IOS** - Wait until Shawn addresses the XCode TODO before testing

* **TEST_LINUX**
1. start Keyman configuration with: `LANGUAGE=mrt_NG km-config`

* **TEST_WINDOWS**
